src/main/java/de/uni_tuebingen/ub/nppm/db/SelektionDB.java

```diff
@@ -72,6 +80,54 @@ static public SelektionBezeichnung getByBezeichnung(String selektion, String Bez
         }
     }
 
+    public static String getBezeichnungByID(String id, String tabelle) throws Exception {
+        String result = "";
+
+        try ( Session session = getSession()) {
+            session.getTransaction().begin();
+
+            String sql = "SELECT Bezeichnung FROM " + tabelle + " WHERE ID= :ID";
+            NativeQuery query = session.createNativeQuery(sql);
+            query.setParameter("ID", id);
+            result = (String) query.getSingleResult();
+            session.getTransaction().commit();
+        }
+        return result;
+    }
+
+    public static String getProvenanceSourceByID(String id, String tabelle) throws Exception {
```

Comment:
Isn't this basically exactly the same as AbstractBase.getProvenanceSource()?

Reply:
Yes i have removed the duplicate